fe/lib/components/LocationSuggest/queries.ts

```diff
@@ -0,0 +1,66 @@
+import { gql } from 'apollo-boost';
+
+const LOCATION_ATTRIBUTES = gql`
+  fragment locationAttributes on Location {
+    id {
+      value
+    }
+    name
+    contextualName
+    countryCode
+  }
+`;
+
+const NESTED_LOCATION_ATTRIBUTES = gql`
+  fragment nestedLocationAttributes on Location {
+    ...locationAttributes
+    parent {
+      ...locationAttributes
+      parent {
+        ...locationAttributes
+        parent {
+          ...locationAttributes
+          parent {
+            ...locationAttributes
+            parent {
+              ...locationAttributes
```

Comment:
Just wondering, is there any reasoning behind the level we go to here?

Reply:
No major rationale behind this. For a location of "Melbourne" It has 4 nested parents that it can resolve. here it's an arbitrary amount to cover our bases, I can reduce it down to 4/ the deepest known relation

> Melbourne > CBD & Inner Suburbs > Melbourne >  Victoria > Australia

Reply:
Nah this is fine, I was just wondering whether we were tracking some standard around subdivision levels that we could document